hoot-core/src/main/cpp/hoot/core/criterion/AreaCriterion.cpp

```diff
@@ -55,6 +55,15 @@ bool AreaCriterion::isSatisfied(const Tags& tags, const ElementType& elementType
     return false;
   }
 
+  // Putting this check here to shortcircut the criterion. If the tag is explicitly set to flase other tests don't matter.
+  if (tags.isFalse("area"))
```

Comment:
What ended up being the need for this change? `PolyToWayGeoModifierAction` doesn't use `AreaCriterion` at all.

Reply:
True but line 287 in `OgrWriter.cpp` has a call to `ElementToGeometryConverter` and this calls `AreaCriterion` 
Without the change, the result from GeometryModifierOp  or the Criterion+Visitor get wiped out when creating an element from the ways.